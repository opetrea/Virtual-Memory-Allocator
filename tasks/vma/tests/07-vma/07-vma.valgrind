==91849== Memcheck, a memory error detector
==91849== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91849== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91849== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91849== Parent PID: 91848
==91849== 
==91849== 
==91849== HEAP SUMMARY:
==91849==     in use at exit: 0 bytes in 0 blocks
==91849==   total heap usage: 11 allocs, 11 frees, 8,416 bytes allocated
==91849== 
==91849== All heap blocks were freed -- no leaks are possible
==91849== 
==91849== For lists of detected and suppressed errors, rerun with: -s
==91849== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
