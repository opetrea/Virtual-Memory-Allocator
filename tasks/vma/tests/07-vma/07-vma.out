The allocated address is outside the size of arena
The allocated address is outside the size of arena
Total memory: 0x2 bytes
Free memory: 0x1 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x1 - 0x2
Miniblock 1:		0x1		-		0x2		| RW-
Block 1 end
